README.md

```diff
@@ -69,6 +69,8 @@ render() {
   // Supported inline styles: https://github.com/facebook/draft-js/blob/master/docs/Advanced-Topics-Inline-Styles.md
   // Supported block types: https://github.com/facebook/draft-js/blob/master/docs/Advanced-Topics-Custom-Block-Render.md#draft-default-block-render-map
   const toolbarConfig = {
+    hideLinkButtons: true,
+    hideBlockTypeDropdown: false,
```

Comment:
Actually, I'd prefer to specify what's included, rather than what's excluded:
```js
const toolbarConfig = {
  display: ['INLINE_STYLE_BUTTONS', 'BLOCK_TYPE_DROPDOWN', 'BLOCK_TYPE_BUTTONS'],
  INLINE_STYLE_BUTTONS: [ ... ],
  ...
}
```
The advantage is that it lets us specify order also!